tests/integration/backup_table_tests/test_backup_table_option.py

```diff
@@ -71,6 +71,7 @@ def check_backup_param_template(self, test_table_name, backup_is_expected):
         # Use raw DDL statement to confirm backup is set correctly on new table
         with open('target/run/test/models/{}.sql'.format(test_table_name), 'r') as ddl_file:
             ddl_statement = ddl_file.readlines()
+
```

Comment:
Should there be changes in this test file at all?

Reply:
honestly not sure why the line break happened not sure i've ever opened this file

Reply:
I just commented again and then realized that I had this comment already so sorry about that. 

We don't want these changes in there if we don't mean to have them occur. You should go to the file and undo them so they don't show up in the PR and are not merged in. For this, since it's a smaller change, just go to the file in question and remove the chanegs made (i.e. remove the newline, fix the file ending) and do a commit. The PR should show the file changes go away then. If you need help, just let me know

Reply:
I just commented again and then realized that I had this comment already so sorry about that. 

We don't want these changes in there if we don't mean to have them occur. You should go to the file and undo them so they don't show up in the PR and are not merged in. For this, since it's a smaller change, just go to the file in question and remove the chanegs made (i.e. remove the newline, fix the file ending) and do a commit. The PR should show the file changes go away then. If you need help, just let me know